src/contexts/options.py

```diff
@@ -0,0 +1,225 @@
+import pygame
+
+import sys
+import configuration.config as config
+import color
+import json
+from .base import Context
+
+TEXT_INCREMENT = 60
+RESERVED_KEYS = ["esc", "enter"]
+screen_w, screen_h = config.SCREEN_RESOUTION
+
+
+class ConfigPlayerContext(Context):
+    def __init__(self, drawer):
+        super(ConfigPlayerContext, self).__init__(drawer)
+
+    def draw_options(self):
+        self.drawer.fill(color.BLACK)
+        try:
+            configuration = json.load(open(config.OPTIONS_FILE))
+        except Exception as e:
+            print(e)
+
+        font = pygame.font.Font(config.FONT_FILE, 40)
+        # List used to track which options to set and which keys are being used
+        list_of_options = []
+        list_of_used_keys = ["return", "escape"]
+        width_pos = (screen_w / 100) * 5
+        # Drawing the options and their respective cubes
+        for count, player in enumerate(configuration):
+            height_pos = 50
+            width_pos = ((screen_w / 10) // len(configuration)) + (screen_w / 100) * ((100 / len(configuration)) * count)
+            option_enumeration = 0
+            pygame.draw.line(
+                self.drawer._surface,
+                color.WHITE,
+                ((screen_w / len(configuration) - 10) * count + 1, 50),
+                ((screen_w / len(configuration) - 10) * count + 1, screen_h)
+            )
+            list_of_options.append([])
+            for setting in configuration[player]:
+
```

Comment:
não adicione esse espaço depois do for. Corrija isso em todos os arquivos modificados.